src/shared/codelens/localLambdaRunner.ts

```diff
@@ -247,12 +247,20 @@ export class LocalLambdaRunner {
         await command.execute()
 
         if (this.localInvokeParams.isDebug) {
-            await waitForDebugPort({
+            const isPortOpen: boolean = await waitForDebugPort({
```

Comment:
What causes `waitForDebugPort` to fail? Should that failing just straight-up cancel the debug session?

Reply:
Waiting for port has a configurable timeout limit. It fails if the port is not available within that time.

Reply:
Currently the system throws an Error, which we don't suitably handle, so this is a better wrapper around that.